go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/medik8s/fence-agents-remediation
 
-go 1.20
+go 1.22
```

Comment:
Update the Golang version in https://github.com/medik8s/fence-agents-remediation/blob/main/.ci-operator.yaml as well for OCP CI